lib/options.js

```diff
@@ -80,6 +80,7 @@ const optionList = [
   'scripts',
   'separator',
   'static',
+  'static_dirs',
```

Comment:
This should also be`staticDirs` I think (or at least I prefer this for consistency).

Reply:
:+1: